### PR TITLE
[GH-622] Bump version to v3.0.0

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
Bump version to v3.0.0

#### Ticket Link
#622

### CODE REVIEW

**Review by levb — APPROVED** [review]

File: plugin.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "support_url": "https://github.com/mattermost/mattermost-plugin-jira/issues",
   "release_notes_url": "https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v2.4.0",
```
Comment (hanzei) [inline plugin.json:7]: This need up get updated

**Review by hanzei — CHANGES_REQUESTED** [review]

**Review by hanzei — APPROVED** [review]
:rocket:

**Review by aaronrothschild — APPROVED** [review]
LGTM

**Comment by DHaussermann** [comment]
At @hanzei  can you please help? 
The only PR added to master should be https://github.com/mattermost/mattermost-plugin-jira/pull/681 
I can't pull it up into this branch.

**Comment by hanzei** [comment]
@DHaussermann Conflict resolved

**Review by DHaussermann — APPROVED** [review]
- Ran through end 2 end test for all precasting Jira 2.4 functionality
- Ran through end 2 end test for new Jira V3 functionality
- Tested Jira migration of Cloud  and server instanced - No issues found
- Tested Jira denigrate with both reinstall  Jira 2.4 path and the re-enable Jira 3 path
- Tested last change for min server version
- Version correctly shown as `3.0.0`

**Comment by DHaussermann** [comment]
@jfrerich Let me know once we have a version in the repo. 
We can then have it applied to community and create a marketplace PR 🎉

**Pull request merged**